Extensions/Multiplayer/messageManager.ts

```diff
@@ -1,5 +1,12 @@
 namespace gdjs {
   const logger = new gdjs.Logger('Multiplayer');
+  /* Set to true to see message logs and ease debugging. */
```

Comment:
Alternatively, you should be able to do:

```
const debugLogger = new gdjs.Logger('Multiplayer - Debug');
// Comment this to see message logs and ease debugging:
gdjs.Logger.getDefaultConsoleLoggerOutput().discardGroup('Multiplayer - Debug');
```

and in the rest of the file, use debugLogger for debug messages. And use `logger` as usual for the important Multiplayer logs.

Reply:
went for 1 declaration of 2 loggers in the main multiplayerTools and imported them where I need them

Reply:
Not a big fan of this unfortunately because: 
- this pollutes gdjs with yet another variable `gdjs.multiplayerDebugLogger` + you risk someone re-declaring it/erasing it/messing with it. Generally speaking, logs are something you must be 100% confident in. 
- Worse, someone could use it for logging something else unrelated to multiplayer.
- this makes it harder to grep `logger.info` in the codebase.
- this makes it harder to scan the top of the file and understand "which loggers this file is using" just by reading a few lines.
- personal taste, but it's just too verbose. gdjs.multiplayerDebugLogger... omg of course I know it's multiplayer, I'm in a multiplayer file. :) 

Logger are lightweight. Create one or more in each file you need the.

Reply:
hmm ok